runtime/jcl/common/mgmtosext.c

```diff
@@ -79,6 +79,33 @@ Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getProce
 	return (0 == rc)? (jlong) size: (jlong) -1;
 }
 
+jdouble JNICALL
+Java_com_ibm_lang_management_internal_ExtendedOperatingSystemMXBeanImpl_getSystemCpuLoadImpl(JNIEnv *env, jobject instance) {
+	PORT_ACCESS_FROM_ENV(env);
+   OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
```

Comment:
Inconsistent indentation.

Reply:
Fixed in [force-push](https://github.com/eclipse/openj9/compare/e200fcd0089727bb8c11ffa2d51bc48880865951..a15fd0a5f1583c638085002e198a4ef9732245cf).